services/kaptain/1.3.0/kaptain.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kaptain
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: kaptain
+      sourceRef:
+        kind: HelmRepository
+        name: kaptain-helm-mirror
+        namespace: kommander-flux
+      version: 1.3.0-dev
```

Comment:
should we set the upcoming release version here? `1.3.0`?

Reply:
This is the values from https://github.com/mesosphere/kudo-kubeflow/blob/3c1a181f6852159ea3ef6b0d3b5d2ef45241aae0/kudo-kubeflow/helm/chart/Chart.yaml#L3

But in general, it will have to be adjusted once the release process is of the kaptain chart is provided. I would say, let's keep it as it is and revisit it when we start working on the release process.